### PR TITLE
Y2K-1253 Team purpose screen

### PR DESCRIPTION
**Pull request by Daniel-Wang**
Screen Shot 2020-02-21 at 3 07 42 PM

### CODE REVIEW

**Comment by Daniel-Wang** [comment]
@buoyad did we want to do the new modal in a new file, right now I've just added it into the existing one

**Comment by buoyad** [comment]
@Daniel-Wang yeah it can go in a new file in `new-team/wizard`

File: shared/teams/new-team/index.tsx [hunk 1]
```diff
@@ -98,6 +153,14 @@ const CreateNewTeam = (props: Props) => {
 }
 
 const styles = Styles.styleSheetCreate(() => ({
+  body: Styles.platformStyles({
+    common: {
+      ...Styles.padding(Styles.globalMargins.small),
+      backgroundColor: Styles.globalColors.blueGrey,
+    },
+    isElectron: {minHeight: 326},
```
Comment (buoyad) [inline shared/teams/new-team/index.tsx]: Add border radius on the bottom left+right

File: shared/teams/new-team/index.tsx [hunk 2]
```diff
@@ -35,6 +37,59 @@ const CreateNewTeam = (props: Props) => {
...
+          direction="vertical"
+          fullWidth={true}
+          style={styles.body}
+          gap={Styles.isMobile ? 'xsmall' : 'tiny'}
+        >
+          <Kb.Text type="BodySemibold">What do you need a team for?</Kb.Text>
+          <Kb.RichButton
+            description="A small group of people, with no initial need for channels."
+            icon="icon-teams-type-squad-64"
+            onClick={() => console.log('hi')}
```
Comment (buoyad) [inline shared/teams/new-team/index.tsx]: add `TODO`s and remove logs

**Pull request merged**